pkg/volume/flocker/plugin_test.go

```diff
@@ -28,15 +30,17 @@ import (
 
 const pluginName = "kubernetes.io/flocker"
 
-func newInitializedVolumePlugMgr() volume.VolumePluginMgr {
+func newInitializedVolumePlugMgr(t *testing.T) (volume.VolumePluginMgr, string) {
 	plugMgr := volume.VolumePluginMgr{}
-	plugMgr.InitPlugins(ProbeVolumePlugins(), volume.NewFakeVolumeHost("/foo/bar", nil, nil))
-	return plugMgr
+	dir, err := ioutil.TempDir("", "flocker")
+	assert.NoError(t, err)
+	plugMgr.InitPlugins(ProbeVolumePlugins(), volume.NewFakeVolumeHost(dir, nil, nil))
+	return plugMgr, dir
```

Comment:
I wonder if there is a better way of getting this `dir` than returning it here (it's not being used in 3 out of 4 functions). Perhaps with the builder created in line 212? There is no problem doing the cleanup there because the function `SetupAt` was not called yet.